src/Resources/app/storefront/src/js/filter-range.plugin.js

```diff
@@ -0,0 +1,212 @@
+import FilterRangePlugin from 'src/plugin/listing/filter-range.plugin';
+import '../../node_modules/nouislider/distribute/nouislider.css';
+import deepmerge from 'deepmerge';
+import DomAccess from 'src/helper/dom-access.helper';
+
+const noUiSlider = require('@nouislider');
+
+export default class FlFilterRangePlugin extends FilterRangePlugin {
+
+  static options = deepmerge(FilterRangePlugin.options, {
+    sliderContainer: '.fl--range-slider',
+  });
+
+  init() {
```

Comment:
On pages where Findologic is not active, the default Shopware 6 price filter should not be overridden. This causes the following issue:

![image](https://user-images.githubusercontent.com/20555112/110462247-b6998780-80d0-11eb-8d61-d5310afca5d7.png)

![image](https://user-images.githubusercontent.com/20555112/110462388-ec3e7080-80d0-11eb-82d5-41d9a95ee71f.png)

Reply:
Using the slider and then clicking the Reset button to reset the filter, will properly update the input fields, but not the range knobs:

![image](https://user-images.githubusercontent.com/20555112/110465225-71775480-80d4-11eb-820a-1e3faaafe8d1.png)